### PR TITLE
Article setup

### PR DESCRIPTION
**Pull request by meredith-myers**
@rogen-code @cinziaborello @JacksonTodd93 
Hi all, here's the PR with the Article data gen implemented as well as the Article API requests implemented. I tried to make it work so that I could filter through the Restaurant tags, pass each as an argument to the Articles query, and return those Articles; however, I ran into some timing/order issues, so for the time being I'm keeping them separate and then plan on working through that once the client side is finalized.

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -71,4 +73,51 @@ for (restaurantCount = 0; restaurantCount < 100; restaurantCount++) {
...
+  }
+
+  return articleData;
+}
+
+for (let articleCount = 0; articleCount < 100; articleCount++) {
+  let articleObject = generateIndividualArticleData();
+
+  let columns = `title, image, url, tags`;
+  let values = '"' + articleData['articleTitle'] + '", ' + '"' + articleData['articleImage'] + '", ' + '"' + articleData['articleUrl'] + '", ' + '"' + articleData['articleTags'] + '"'
```
Comment (ryancmoniz) [inline database/seed.js:113]: Could we use template strings here instead of concatenation to save some line space?

File: database/index.js [hunk 2]
```diff
@@ -23,6 +23,19 @@ let getRestaurantInfo = function(restaurantId, callback) {
...
+    if (error) {
+      console.log('Error at getRestaurantArticles in database index file');
+      callback(error, null);
+      return;
+    }
+    console.log('Success at getRestaurantArticles in database index file');
+    callback(null, results);
+  });
+};
+
```
Comment (ryancmoniz) [inline database/index.js:37]: Once you are confident your code is behaving correctly, you can get rid of console.logs that are just telling you what happened inside a function.

File: server/index.js [hunk 3]
```diff
@@ -6,13 +6,31 @@ const port = 6070
...
+
   db.getRestaurantInfo(restaurantId, function(error, data) {
     if (error) {
-      console.log('Error at server GET request');
+      console.log('Error at server/restaurants GET request');
+    } else {
+      console.log('Successful server/restaurants GET request');
+      console.log(data);
+      res.send(data);
+    }
```
Comment (ryancmoniz) [inline server/index.js:22]: Same comment as before, too many console.logs eventually just fill your console with meaningless information. Also, if I make a request to your api and there is an error, what happens? Does the app crash? As a user how do I know the API is working? Do you send anything back to me letting me know about the _status_ of my request?

File: server/index.js [hunk 4]
```diff
@@ -6,13 +6,31 @@ const port = 6070
...
+
+app.get('/api/articles/:id', function(req, res) {
+  db.getRestaurantArticles(articleTag, function(error, data) {
+    if (error) {
+      console.log('Error at server/articles GET request');
     } else {
-      console.log('Successful server GET request');
+      console.log('Successful server/articles GET request');
       console.log(data);
       res.send(data);
```
Comment (ryancmoniz) [inline server/index.js:35]: Here as well. If the article I search doesn't exist, how would I know?

File: database/seed.js (hunk 1 above)
Comment (meredith-myers) [reply database/seed.js:113]: Fantastic suggestion, Ryan! So much cleaner, shorter, and more readable.

File: server/index.js (hunk 3 above)
Comment (meredith-myers) [reply server/index.js:22]: In this situation, that component simply wouldn't render if there are no relevant articles to display for that particular restaurant. I hadn't thought about sending a message to the user about the request, so I'll definitely keep that in mind going forward! And will be sure to remove those console.logs once everything's closer to being final.

**Pull request merged**